.github/workflows/golangci-lint.yaml

```
@@ -17,4 +17,4 @@ jobs:
       uses: golangci/golangci-lint-action@v3
       with:
         version: v1.55.2
```

```suggestion
        version: v1.58.2
```

And also bump following

https://github.com/kubernetes-sigs/kwok/blob/main/hack/update-go-lint.sh
https://github.com/kubernetes-sigs/kwok/blob/main/hack/verify-go-lint.sh